src/App.tsx

```diff
@@ -1,24 +1,203 @@
-/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
+import {
+  FC,
+  useEffect,
+  useMemo,
+  useState,
+} from 'react';
+import {
+  USER_ID as id,
+  getTodos,
+  postTodo,
+  deleteTodo,
+} from './api/todos';
+import Footer from './Components/Footer/Footer';
+import Header from './Components/Header/Header';
+import TodoList from './Components/TodoList/TodoList';
+import Notification from './Components/Notification/Notification';
+import Loader from './Components/Loader/Loader';
+import { Todo } from './types/Todo';
+import { Error, ErrorMessage } from './types/Error';
+import { FilterType } from './types/FilterType';
+import { filterTodoList, getMountCompletedTodos } from './utils/filterTodoList';
 
-const USER_ID = 0;
+export const App: FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [amountCompletedTodos, setAmountCompletedTodos] = useState(0);
```

Comment:
looks like no need to create state for counting completed todos.
better to make it in use memo based on todos.
whenewer changed todos useMemo will work and count completed todos